src/control/server/init/setup_spdk.sh

```diff
@@ -17,9 +17,11 @@ else
 	PCI_WHITELIST="$_PCI_WHITELIST" NRHUGE="$_NRHUGE" \
 	TARGET_USER="$_TARGET_USER" "$scriptpath"
 
-	chmod 777 /dev/hugepages
-	chmod 666 /dev/uio*
-	chmod 666 /sys/class/uio/uio*/device/config
-	chmod 666 /sys/class/uio/uio*/device/resource*
+	if [[ "$_TARGET_USER" != "root" ]]; then
+		chmod 777 /dev/hugepages
```

Comment:
I know we're planning on changing things with the privileged binary work, but in the meantime this seems like a security risk. These commands are giving blanket access to the devices to all users. Wouldn't it be better to change ownership to the target user?

Reply:
we can change that but seems like a separate piece of work, this PR just stops the commands from being executed when running as root (as queried by Johann)

Reply:
If I had seen this before, I would have raised my concerns before. Now that I've seen it, I can't in good conscience allow a serious security problem to be perpetuated because it's not in scope for a PR.

We're removing all protection from these devices with these commands, allowing any user to read from or write to them. This is at a minimum a large potential security/privacy risk, and could lead to data loss.